### PR TITLE
Upgrade @kiwicom/account and pass bridge function

### PR DESCRIPTION
**Pull request by tbergq**
Cleanup code on ios since it was broken
And move it to RNModules

### CODE REVIEW

**Comment by tbergq** [comment]
@RobinCsl, if you want to test that the method is there, the easiest way is to just add `console.warn(NativeModules.RNAccountManager.accountDeleted);` to `WithAccountStack.js`

@sjavora , there were some syntax errors, so I had to fix them. I also moved to Bridge module to `RNModules` where the rest of the bridge modules lives. Please verify that I did not break anything.

**Comment by tbergq** [comment]
also, @RobinCsl , I didn't test on android yet. I want to do that after this one has been merged.

File: ios/RNModules/RNAccountManager/RNAccountManager/RNAccountManager.m [hunk 1]
```diff
@@ -6,12 +6,19 @@
...
 
-#import "RNKiwiAccountUpdater.h"
 
-@implementation RNKiwiAccountUpdater
 
+@implementation RNAccountManager
+
+RCT_EXPORT_MODULE();
+
++ (BOOL)requiresMainQueueSetup {
```
Comment (sjavora) [inline ios/RNModules/RNAccountManager/RNAccountManager/RNAccountManager.m]: On the native side, this is doing nothing here that requires the main queue - why was this added?

File: ios/RNModules/RNAccountManager/RNAccountManager/RNAccountManager.h [hunk 2]
```diff
@@ -9,8 +9,9 @@
 @import Foundation;
 #import "React/RCTBridgeModule.h"
 
+@protocol RCTBridgeModule;
 RCT_EXTERN const NSNotificationName _Nonnull RNKiwiAccountDeletedNotification;
 
-@interface RNKiwiAccountUpdater : NSObject <RCTBridgeModule>
+@interface RNAccountManager : NSObject <RCTBridgeModule>
```
Comment (sjavora) [inline ios/RNModules/RNAccountManager/RNAccountManager/RNAccountManager.h]: I specifically named this class so that it wouldn't be a "manager". Why would you change this?

File: ios/RNModules/RNAccountManager/RNAccountManager/RNAccountManager.m (hunk 1 above)
Comment (tbergq) [reply ios/RNModules/RNAccountManager/RNAccountManager/RNAccountManager.m]: I can try running without it, I really just saw that most of our modules have this

**Comment by tbergq** [comment]
> What exactly was not building? I tried it before submitting and had no errors.

It was missing an `;`  in `RNKiwiViewController.m` line 240, and `NSNotificationName ` was placed before the import of the `.h` file, so it was not defined. 

And it did not build for me 😊

File: ios/RNModules/RNAccountManager/RNAccountManager/RNAccountManager.h (hunk 2 above)
Comment (tbergq) [reply ios/RNModules/RNAccountManager/RNAccountManager/RNAccountManager.h]: So that we have the same name for android and ios

File: ios/RNModules/RNAccountManager/RNAccountManager/RNAccountManager.m (hunk 1 above)
Comment (tbergq) [reply ios/RNModules/RNAccountManager/RNAccountManager/RNAccountManager.m]: Seems it was not necessary, so I removed it again. Thanks.

**Comment by sjavora** [comment]
Oh yeah, I was building a different target - that explains why those build errors didin't show up /facepalm

File: ios/RNModules/RNAccountManager/RNAccountManager/RNAccountManager.h [hunk 3]
```diff
@@ -9,8 +9,9 @@
 @import Foundation;
 #import "React/RCTBridgeModule.h"
 
+@protocol RCTBridgeModule;
```
Comment (sjavora) [inline ios/RNModules/RNAccountManager/RNAccountManager/RNAccountManager.h]: This shouldn't be necessary - it is defined in the import above

Comment (tbergq) [reply ios/RNModules/RNAccountManager/RNAccountManager/RNAccountManager.h]: Yes, looks like you are right, I will remove it

Comment (tbergq) [reply ios/RNModules/RNAccountManager/RNAccountManager/RNAccountManager.h]: @sjavora Removed
